.travis.yml

```diff
@@ -1,16 +1,21 @@
 language: python
 python:
   - "3.6"
-services:
-  - postgresql
+
+addons:
+  postgresql: "9.4"
```

Comment:
Why are we using postgres `9.4`? I suggest we use `9.6`

Reply:
Ok. I have used 9.6